reset-password.py

```diff
@@ -109,4 +109,4 @@ def validate_pin(pin):
         email.dump_to_file(config)
         raise
 
-   
+    print "Password successfully reset for user {}".format(user.name)   
```

Comment:
The script is run interactively by the helpdesk people, so they will see this printed to the console, that way they are reassured that it worked.